### PR TITLE
[document_repository] add tooltip 

### PR DESCRIPTION
**Pull request by kongtiaowang**
to solve : #4778
When you hover the category list, it will show the tooltip

### CODE REVIEW

File: modules/document_repository/jsx/childTree.js [hunk 1]
```diff
@@ -42,17 +70,21 @@ class ChildTree extends Component {
...
+            <div
+              style={headerRow}
               key={index}
               className="pointer"
-              style={spanStyle}
             >
-              <i className="fa fa-folder"></i>
-              <span style={spanStyle}>
+              <i className="fa fa-folder" style={iconStyle}></i>
+              <div style={spanStyle} id="tooltip">
```
Comment (driusan) [inline modules/document_repository/jsx/childTree.js]: `id`s need to be unique across the entire page. I think this component is used multiple times on the same page, so a hardcoded "tooltip" won't work.

Comment (nicolasbrossard) [reply modules/document_repository/jsx/childTree.js]: This PR does not work on my sandbox (no tooltip shown...)

**Comment by kongtiaowang** [comment]
Screen Shot 2019-06-18 at 3 25 25 PM

**Comment by kongtiaowang** [comment]
@nicolasbrossard Could u test it again? it only shows when the comment is not empty.

**Comment by johnsaigle** [comment]
Comments for categories do not appear for me.

**Pull request merged**